packages/manager/src/constants/API_ENDPOINTS.ts

```diff
@@ -0,0 +1,62 @@
+import { APPLICATION_MODE } from "./APPLICATION_MODE";
+
+type APIEndpoints = {
+	PrismicWroom: string;
+	PrismicAuthentication: string;
+	PrismicModels: string;
+	PrismicUser: string;
+	AwsAclProvider: string;
+};
+
+export const API_ENDPOINTS: APIEndpoints = (() => {
+	switch (process.env.SM_ENV) {
+		case APPLICATION_MODE.Development: {
+			const apiEndpoints = {
+				PrismicWroom: process.env.wroom_endpoint,
+				PrismicAuthentication: process.env.authentication_server_endpoint,
+				PrismicModels: process.env.customtypesapi_endpoint,
+				PrismicUser: process.env.user_service_endpoint,
+				AwsAclProvider: process.env.acl_provider_endpoint,
+			};
+
+			const missingAPIEndpoints = Object.keys(apiEndpoints).filter((key) => {
+				return !apiEndpoints[key as keyof typeof apiEndpoints];
+			});
+
+			if (missingAPIEndpoints.length > 0) {
+				console.error(
+					`You are running Slice Machine in development mode where API endpoints are configured via environment variables. The following endpoints were not configured: ${missingAPIEndpoints.join(
+						", ",
+					)}. Configure them before continuing.`,
+				);
+
+				process.exit(1);
```

Comment:
Good point! I don't think we test different environments since the different environments are themselves only used for testing.

If you have thoughts on how this should be tested, please share! 🙂

(I suggest we don't perform a unit test and import this file directly. We should try to do integration tests instead.)